core/src/Streamly/Internal/Data/Parser.hs

```diff
@@ -1200,7 +1200,9 @@ takeP i p = D.toParserK $ D.takeP i $ D.fromParserK p
 -- container @t@ using the fold @f@. Fails if the input ends or any of the
 -- parsers fail.
 --
--- This is same as 'Data.Traversable.sequence' but more efficient.
+-- This is same as 'Data.Traversable.sequence' but more efficient. An
+-- even more efficient implementation can use ParserD type Parser in
+-- the SerialT stream.
```

Comment:
I am not sure if it is more efficient as there is no fusion, I guess it would be the same. But it can use a stream of parsers rather than a traversable container, so is more general.

Reply:
Removed the incorrect statement.